app/presenters/tree_builder_miq_action_cat.rb

```diff
@@ -0,0 +1,46 @@
+class TreeBuilderMiqActionCat < TreeBuilder
+  has_kids_for Classification, [:x_get_tree_classification_kids]
+
+  private
+
+  # Maybe we should define NodeBuilder inside this class?
+  def node_builder
+    TreeNodeBuilderMiqActionCat
+  end
+
+  def initialize(name, type, sandbox, build = true, tenant_name)
+    @tenant_name = tenant_name
+    super(name, type, sandbox, build)
+  end
+
+  def tree_init_options(_tree_name)
+  {:expand        => true,
+   :lazy          => false}
+  end
+
+  def set_locals_for_render
+    locals = super
+    locals.merge!(:id_prefix   => "cat_tree",
+                  :click_url => "/miq_policy/action_tag_pressed/",
+                  :onclick   => "miqOnClickTagCat")
+  end
+
+  def root_options
+    title = @tenant_name
+    [title, title, "tag.png"]
```

Comment:
completely fine, but the usual pattern for these things is `[t = @tenant_name, t, "tag.png"]`